api/alembic/versions/2169aa76dfea_add_haida_gwaii_zone_to_cofc.py

```diff
@@ -0,0 +1,125 @@
+
+
+def get_fire_centre_id(conn, name: str):
+    res = conn.execute(sa.text('SELECT id FROM fire_centres WHERE name = \'{}\''.format(name)))
+    results = res.fetchall()
+    return results[0][0]
+
+
+def get_zone(conn, name: str, fire_centre_id: int):
+    res = conn.execute(
+        sa.text('SELECT * FROM planning_areas WHERE name = \'{}\' AND fire_centre_id = {}'.format(name, fire_centre_id)))
+    results = res.fetchall()
+    return results[0]
+
+
+def get_planning_weather_station(conn, station_code: int, planning_area_id: int):
+    res = conn.execute(sa.text(
+        'SELECT * FROM planning_weather_stations WHERE station_code = {} and planning_area_id = {}'.format(station_code, planning_area_id)))
+    results = res.fetchall()
+    return results[0]
+
+
+def get_fuel_type_id(conn, fuel_type_abbrev: str):
+    res = conn.execute(sa.text('SELECT id FROM fuel_types WHERE abbrev = \'{}\''.format(fuel_type_abbrev)))
+    results = res.fetchall()
+    return results[0][0]
+
+
+def increase_value_order_of_appearance(conn, fire_centre_id: int, from_index: int):
+    # due to constraint on unique value of (fire_centre_id, order_of_appearance_in_list), have to
+    # modify values of order_of_appearance_in_list one at a time, otherwise error is thrown
+    res = conn.execute(
+        sa.text('SELECT MAX(order_of_appearance_in_list) FROM planning_areas WHERE fire_centre_id = {}'.format(fire_centre_id)))
+    max_value_order = res.fetchall()[0][0]
+    index = max_value_order
+    while index > from_index:
+        conn.execute(sa.text('UPDATE planning_areas SET order_of_appearance_in_list = order_of_appearance_in_list + 1 WHERE fire_centre_id = {} AND order_of_appearance_in_list = {}'.format(fire_centre_id, index)))
+        index -= 1
+
+
+def decrease_value_order_of_appearance(conn, fire_centre_id: int, to_index: int):
+    # due to constraint on unique value of (fire_centre_id, order_of_appearance_in_list), have to
+    # modify values of order_of_appearance_in_list one at a time, otherwise error is thrown
+    res = conn.execute(
+        sa.text('SELECT MAX(order_of_appearance_in_list) FROM planning_areas WHERE fire_centre_id = {}'.format(fire_centre_id)))
+    max_value_order = res.fetchall()[0][0]
+    index = to_index
+    while index <= max_value_order:
+        conn.execute(sa.text('UPDATE planning_areas SET order_of_appearance_in_list = order_of_appearance_in_list - 1 WHERE fire_centre_id = {} AND order_of_appearance_in_list = {}'.format(fire_centre_id, index)))
+        index += 1
+
+# ------ end of helper functions ----
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    cofc_id = get_fire_centre_id(conn, 'Coastal Fire Centre')
+    fraser_zone = get_zone(conn, 'Fraser Zone', cofc_id)
+    fraser_zone_id = fraser_zone[0]
+    m2_25_id = get_fuel_type_id(conn, 'M2 25%')
+
+    # 1. Increase order_of_appearance_in_list by 1 for all planning areas in COFC with order value > 1
+    increase_value_order_of_appearance(conn, cofc_id, 1)
```

Comment:
If Haida Gwaii should have order 2, shouldn't we just bump orders for areas where order is >= 2?

Reply:
that is essentially what's happening, I'm just using `>1` rather than the equivalent `>=2` (it saves one keystroke don'tcha know?! 😉 )

Reply:
Oh my bad I see now! Thanks. Maybe we could pull this out of a migration at some point to expose for users to adjust in the UI.